pkgs/os-specific/linux/firmware/fwupd/default.nix

```diff
@@ -257,6 +258,10 @@ let
         contrib/generate-version-script.py \
         meson_post_install.sh \
         po/test-deps
+
+      substituteInPlace data/installed-tests/fwupdmgr-p2p.sh \
+        --replace "/usr/libexec/fwupd/fwupd" $out/libexec/fwupd/fwupd \
```

Comment:
The FHS path should be fixed upstream. Could you please try to make a PR?

Reply:
What should it be?

/usr/libexec/fwupd/fwupd seems to comply with https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s07.html

Reply:
It should be `@libexecdir@/fwupd/fwupd` and substituted using Mesonʼs `configure_file` function.

Reply:
Ah, that makes sense. I'm not familiar with Meson so have raised this as https://github.com/fwupd/fwupd/issues/4360.